services/meta/client_test.go

```diff
@@ -239,8 +239,8 @@ func TestMetaClient_CreateRetentionPolicy(t *testing.T) {
 	rp0 := meta.RetentionPolicyInfo{
 		Name:               "rp0",
 		ReplicaN:           1,
-		Duration:           time.Hour,
-		ShardGroupDuration: time.Hour,
+		Duration:           2 * time.Hour,
```

Comment:
This change was required as tests further down rely on changing the RP duration and it was dropping below the new min enforced floor.  This was the easiest way to fix that test other than rewriting the entire test.